src/nmdc_submission_schema/scripts/export_nmdc_schema.py

```diff
@@ -0,0 +1,16 @@
+import importlib.resources
+
+import click
+
+
+@click.command()
+@click.option('--output-file', '-o', type=click.File('w'), default="-")
+def main(output_file):
```

Comment:
```suggestion
def main(output_file):
    r"""
    Dumps a YAML representation of the NMDC Schema to the specified file.
    """
```

I suggest adding a description to the click command.

https://click.palletsprojects.com/en/stable/documentation/#help-texts

Reply:
Good shout. Thank you